ssz/src/main/java/org/ethereum/beacon/ssz/SSZSchemeBuilder.java

```diff
@@ -36,6 +37,21 @@
        */
       public boolean notAContainer = false;
 
+      /**
+       *  If the field class specifies {@link SSZSerializable#serializeAs()} attribute
+       *  returns the specified class.
+       *  Else returns type value.
+       */
+      public Class<?> getSerializableClass() {
```

Comment:
I'd put this logic somewhere in `SSZSchemeBuilder`
The idea behind these classes was that `SSZField` just describes the scheme of object